src/components/Sitsit.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "../App.css";
+
+import Table from "./Table";
+import { arrayOf, number } from "prop-types";
+import { table } from "../propTypes";
+
+const DEFAULT_NUMBER_OF_TABLES = 2;
```

Comment:
`src/constants/defaults.js` pls, and then take it as a prop?

Reply:
Sure!